pagopa/io-app
# feat(IT Wallet): [SIW-403] Add navigation to qrcode tab bar button

## Short description
This PR enable qrcode navigation to BaseScanScreen.

## How to test
Run the app and try to scan a sample RP qrcode. Add a console.log to check if clientId and authReqUrl are decoded correctly.
